### ISSUE TITLE
Not possible for links to be opened in new tab in inline mode

### ISSUE THREAD

**Issue by bfncs** [ryancramerdesign/InputfieldCKEditor#5]
Hello Ryan,

thanks for this great module in the first place. I just recognized one slight issue that occurs to me, when I use the Inputfield in inline mode:

If i set an anchor to be opened in a new window/tab, this is not persisted to the database. The HTML is correct right after editing (checking with the CKEditor HTML view), but after saving the page this is lost again.

I'm pretty sure, that this is caused by a too strict configuration of the HTMLPurifier module, but didn't have enough time to look further into it until now.

With kind regards,
Marc

Edit: I think this can be fixed by setting `Attr.AllowedFrameTarget` to `array('_target')` before purifying, can't test it at the moment, though,

**Comment by ryancramerdesign** [comment]
Does the issue only occur on inline mode, or in the regular edit mode too?

Thanks,
Ryan

**Comment by bfncs** [comment]
Hi there, I found the time to further investigate into it and it definitely only happens in inline mode. `target="_blank"` is filtered out by HTMLPurifier.

While trying to fix it, I stumbled into a slight bug in MarkupHTMLPurifier: ryancramerdesign/MarkupHTMLPurifier#1. Send you a pull request for both modules now, the one here is relying on the other one to be applied first, making HTMLPurifier configurable.
